src/runtime_functions.js

```diff
@@ -121,7 +121,7 @@ function addFunctionWasm(func, sig) {
   if (!functionsInTableMap) {
     functionsInTableMap = new WeakMap();
     for (var i = 0; i < wasmTable.length; i++) {
-      var item = wasmTable.get(i);
+      var item = wbind(i);
```

Comment:
wbind is a library function .. so using it here in the a runtime function is tricky because we have no way to declare `addFunctionWasm__deps` on it.    Maybe its time to move `addFunctionWasm` to a library function?

Reply:
Writing wbind here seems odd.. especially since its dual is setWasmTableEntry.

How about `getWasmTableEntry` to mirror `setWasmTableEntry`?   Then `getWasmTableEntry` have have two different flavors and `wbind` can just be an alias for `getWasmTableEntry`  in library.js?

If you don't like that idea how about the other way around.. make `$getWasmTableEntry` and alias of wbind?

Reply:
> Maybe its time to move `addFunctionWasm` to a library function?

We have used this mechanism of "runtime depends on library and auto-adds the needed library funcs as always exported" in a number of places, so I'll go with that for now.

Refactoring `addFunctionWasm` to a library function does sound ok, but I'll keep that off from this code to not creep the PR size.